### ISSUE TITLE
Calibration upload fc1d05

### ISSUE THREAD

**Issue by bronger** [lensfun/lensfun#450]
Calibration images were uploaded to the directory that starts with “`fc1d05_`”.

Please read the [workflow description](https://github.com/lensfun/lensfun/blob/master/tools/calibration_webserver/workflow.rst) for further instructions about the calibration.

Additional comments by the uploader:

> Here is my submission. Photos were shot on Sony A7R III (ILCE-7RM3).
> Lens: Sony FE 24-105mm F4 G OSS
> Focal Lengths: 24, 35, 51, 70 and 105mm
> 
> Thanks in advance,
> 
> Tony

**Comment by sarunasb** [comment]
Sony A7R III (ILCE-7RM3) and corrections for FE 24-105mm F4 G OSS lens
are already in calibration database.

-- 
Sarunas Burdulis
Systems Administrator, Dartmouth Mathematics
math.dartmouth.edu/~sarunas

**Comment by TonyInOakville** [comment]
Thanks sarunasb but when I download this file and replace the current one in the ACDSee Lensfun folder, it is totally ignored and the slr-sony.xml file is used instead. Maybe I'm doing something wrong.
[mil-sony.zip](https://github.com/lensfun/lensfun/files/2320979/mil-sony.zip)

**Comment by TonyInOakville** [comment]
I can't figure it out. I've examined the original xml file and the new one containing my camera and lens. I'm not a programmer, but examining the new file I can't see any errors or missing parameters. the camera and lens names are as they appear in the exif data. I've even tried pasting the new camera and lens data into the old file, and again, it does not show up in ACDSee. 
I'll bounce this back to ACDSee support as they are the ones who directed me to Lensfun to get my camera and lens calibrated. 
Thanks

**Comment by junkyardsparkle** [comment]
Actually, it sounds like file names and contents of files might be significant in ACDSee's wonky implementation:
https://www.mu-43.com/threads/raw-conversion.95894/page-4#post-1124898

**Comment by sarunasb** [comment]
Meaning they use XML database only, not the Lensfun itself?...

**Comment by junkyardsparkle** [comment]
Apparently. Possibly because the database itself is less restrictively licensed...

**Comment by TonyInOakville** [comment]
Just to update you guys on this. 
ACDSee support forwarded my problem to their development team. Today I received a new? "mil-sony.xml" file (I did not compare it to the one sarunasb sent me so it could be the copy I forwarded to them). I also received a new "ID_DCRaw.apl" that I copied to the ACDSee plugin folder.
Now everything works correctly and both my camera and lens are recognized in the ACDSee Develop mode.
Thanks for your help.

[SonyA7RM3_24-105mm.zip](https://github.com/lensfun/lensfun/files/2363569/SonyA7RM3_24-105mm.zip)
